### ISSUE TITLE
fedora 31 packages in next-devel

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#463]
Our next-devel stream is based on Fedora 32. There are currently some packages in `next-devel` that are still their f31 counterparts:

```
$ rpm-ostree --repo=./ db list fa91e1bea89e665377ce9a0d993b38dcdfe2bfe623e1f6f5b5c06dbaf87021e6 | grep fc31
 conmon-2:2.0.15-1.fc31.x86_64
 containerd-1.2.6-2.20190627gitd68b593.fc31.x86_64
 coreos-installer-0.1.3-1.fc31.x86_64
 coreos-installer-systemd-0.1.3-1.fc31.x86_64
 slirp4netns-1.0.0-1.fc31.x86_64
 zincati-0.0.9-1.fc31.x86_64
```

- `conmon-2:2.0.15-1.fc31.x86_64`
    - The latest version in f32 stable right now is [conmon-2.0.14-1.fc32](https://bodhi.fedoraproject.org/updates/FEDORA-2020-8e9b996b3c). [conmon-2.0.15-1.fc32](https://bodhi.fedoraproject.org/updates/FEDORA-2020-47f4db8eef) is in testing but the bodhi update has negative karma. 
- `containerd-1.2.6-2.20190627gitd68b593.fc31.x86_64`
    - [containerd-1.3.3-1.fc32](https://bodhi.fedoraproject.org/updates/FEDORA-2020-db5b53c9b0) will be pushed to stable once F32 unfreezes, we can do an override for now
- `slirp4netns-1.0.0-1.fc31.x86_64`
    - The latest version in f32 stable right now is [slirp4netns-0.4.3-5.0.dev.gita8414d1.fc32](https://bodhi.fedoraproject.org/updates/FEDORA-2019-f191b272a8).  [slirp4netns-1.0.0-1.fc32](https://bodhi.fedoraproject.org/updates/FEDORA-2020-228b9254ce) is in testing. We should probably create an override for this.
- `coreos-installer-0.1.3-1.fc31.x86_64`, `coreos-installer-systemd-0.1.3-1.fc31.x86_64`, `zincati-0.0.9-1.fc31.x86_64`
    - @bgilbert @lucab: The next time we spin these packages can we do a fc32 build as well?

**Comment by cgwalters** [comment]
Ohh right, the "Rust packages using Fedora Rust tooling" follow the pattern of being built only in rawhide and tagged in (manually) - what a mess.

Where is that procedure documented again?

**Comment by dustymabe** [comment]
> Where is that procedure documented again?

https://pagure.io/releng/issue/9229

**Comment by lucab** [comment]
Latest Zincati builds (F31 & F32) are in both next-devel and testing-devel (thanks @rfairley for that).

**Comment by jlebon** [comment]
I think we can close this ticket now, right? With https://github.com/coreos/fedora-coreos-config/pull/370, the latest next-devel has no f31 packages. (We actually have the opposite issue, which is that we have some f32 packages in testing-devel, though they're all known quantities -- see https://github.com/coreos/fedora-coreos-config/pull/381).

**Comment by dustymabe** [comment]
fixed in the next stream `32.20200505.1.0` (and newer) and `next-devel` stream `32.20200505.10.1` (and newer)
